{"version":3,"file":"PouchRetryError.js","sourceRoot":"","sources":["../../../../../src/facade-implementations/database/pouch-db-wrapper/errors/PouchRetryError.ts"],"names":[],"mappings":";;;AAAA,MAAa,eAAgB,SAAQ,KAAK;IAGxC;;;;OAIG;IACH,YAAmB,OAAe;QAChC,KAAK,CAAC,OAAO,CAAC,CAAC;QARjB;;;;mBAAgC,iBAAiB;WAAC;IASlD,CAAC;CACF;AAXD,0CAWC"}